api/bibliotheca.py

```diff
@@ -833,7 +833,6 @@ def run_once(self, start, cutoff):
                         exc_info=e
                     )
                 raise e
-            self.timestamp.timestamp = most_recent_timestamp
```

Comment:
This code doesn't work anymore, because `self.timestamp` is now a method. It's no longer necessary because `Monitor.run` takes the return value and sets the `Timestamp` object appropriately.